ci-operator/step-registry/ipi/conf/vsphere/check/ipi-conf-vsphere-check-commands.sh

```diff
@@ -92,14 +97,17 @@ export vsphere_portgroup="${vsphere_portgroup}"
 export vlanid="${vlanid:-unset}"
 export phydc="${phydc:-unset}"
 export primaryrouterhostname="${primaryrouterhostname:-unset}"
+export multizone_portgroup1="${multizone_portgroup1:-unset}"
+export multizone_portgroup2="${multizone_portgroup2:-unset}"
 EOF
 
 # shellcheck source=/dev/null
 source "${SHARED_DIR}/govc.sh"
 
 DATACENTERS=("$GOVC_DATACENTER")
 # If testing a zonal install, there are multiple datacenters that will need to be cleaned up
-if [ ${LEASE_NUMBER} -ge 151 ] && [ ${LEASE_NUMBER} -le 157 ]; then
+# vlanid between 1287 and 1302 will use profile:vsphere-multizone-2
+if [ ${vlanid} -ge 1287 ] && [ ${vlanid} -le 1302 ]; then
```

Comment:
Do we need to consider clean up virtual machines attached to `multizone_portgroup1` and `multizone_portgroup2`? From line130, seems only vms attached to `vsphere_portgroup` will be deleted.

Reply:
Good point, if that is the something missing, I think `VSPHERE_CONNECTED_LEASED_RESOURCE` case also need some fix? cc @sgaoshang

Reply:
And if we don't have plan to hardcode bastion host segment in the future, I guess we can remove bastion from below command, so that leftover bastion host can also be destroyed.

```
jq '.elements[].Path | select((contains("ova") or test("\\bci-segment-[0-9]?[0-9]?[0-9]-bastion\\b")) | not)'
```

Reply:
@jinyunma @WenXinWei @jianlinliu @sgaoshang @liangxia

`if [ ${vlanid} -ge 1287 ] && [ ${vlanid} -le 1302 ]; then` 

Please stop hard coding values. This makes our job managing CI much more difficult.

Reply:
The original segment checking was introduced in `4cf3e8b` commit , if we stop hard coding, so we have to copy the step as a separate step dedicated for multi-zone case?

Reply:
I saw https://github.com/openshift/release/pull/46163 is opened to address this issue, let us discuss in that PR.